pkg/client/httputils.go

```diff
@@ -232,11 +235,16 @@ func (u *FilesUploader) Upload(files []string) error {
 			if returnErr == nil {
 				returnErr = err
```

Comment:
you can improve this line by making `returnErr` a `multierror` and appending each received error to it.